Tutorial on mental health in geosciences (#27)
ICESAT-2HackWeek/website-2024: book/tutorials/mental-health/index.md

* learn individual and community-based approaches to create culture change and normalize mental health conversations
* learn about available resources to find support when navigating mental health challenges

## Summary Description

[Recent data](https://www.hhs.gov/surgeongeneral/priorities/workplace-well-being/index.html#framework) suggest that over 3/4 of people in the U.S. workplace have experienced one or more mental health challenge associated with their workplace. And from our own personal experience, geoscientists face unique challenges in navigating the ups and downs of academic careers, and the grief associated with climate change, among other issues. 

In this session we will begin a conversation about mental health in the geosciences. We will offer resources and follow a [recent framework created by the US Surgeon General](https://www.hhs.gov/sites/default/files/workplace-mental-health-well-being.pdf). If there is sufficient interest we will plan for continued meetings in an effort to build a community of practice around this topic.

This session is optional and we will establish norms that ensure confidentiality for all participants.

## Preparing for the Session

> We invite participants to arrive with a willingness to share stories and listen to their colleagues. This session is intended to be a no-device session, to ensure a distraction-free, immersive experience for all of us. We request all of the interested particpants to familiarize theselves with some our shared norms (above), which will also be provided in printed format during the session. The hackweek team will provide stationary materials to participants for personal note taking! 

Reviewer: ```suggestion
We invite participants to arrive with a willingness to share stories and listen to their colleagues. This session is intended to be a no-device session, to ensure a distraction-free, immersive experience for all of us. We request all of the interested participants to familiarize theselves with some our shared norms (above), which will also be provided in printed format during the session. The hackweek team will provide stationery materials to participants for personal note taking! 
```

We encourage people from all career-stages to join this session!

## Broad format

> Respecting everyone's time committment, we have designed this session to be semi-interactive. The session will be a mix of interview-style interaction between Anthony and Shashank, and include small group discussions between participants in small groups. 

Reviewer: ```suggestion
Respecting everyone's time commitment, we have designed this session to be semi-interactive. The session will be a mix of interview-style interaction between Anthony and Shashank, and include small group discussions between participants in small groups. 
```

## Links to material

> In this section, we provide our initial curated list of resources. We acknowledge that this list is non-exhaustive, and encourage participants to suggest additional relevant resources with which they are familiar with.

Reviewer: ```suggestion
In this section, we provide our initial curated list of resources. We acknowledge that this list is not exhaustive and encourage participants to suggest additional relevant resources with which they are familiar :)```

* Institution based mental health support: Many institutions have resources to support students and employees for their mental health needs. Here we provide examples for University of Washington students and employees as reference, but encourage participants for exploring similar resources at their respective institutions.

>     * Anyone in an immediate crisis situation should call or text 988, or chat on [988Lifeline.org](https://988Lifeline.org).  

Reviewer: ```suggestion
    * Anyone in an immediate crisis situation should call or text 988, or chat on [988Lifeline.org](https://988Lifeline.org).  
    * Everyone in the UW community can access [SAFECAMPUS](https://www.washington.edu/safecampus/) resources for mental health and violence prevention assistance during regular work hours.
```

    * Students: All students at the University of Washington have access to same day crisis counselling through the [Husky Helpline](https://wellbeing.uw.edu/huskyhelpline/). 

>     * More structured support can be accessed through the [peer wellness coaching group](https://livewell.uw.edu/pwc/), [UW Mental health counselling center](https://wellbeing.uw.edu/unit/counseling-center/), and [group therapy programs](https://www.polisci.washington.edu/news/2023/01/19/advertised-groups-winter-2023). 

Reviewer: ```suggestion
    * More structured support can be accessed through the [UW Mental health counseling center](https://wellbeing.uw.edu/unit/counseling-center/), [peer wellness coaching group](https://livewell.uw.edu/pwc/), and [group therapy programs](https://www.polisci.washington.edu/news/2023/01/19/advertised-groups-winter-2023). 
```

    * Services by the Mental Health Counselling Center are all covered by student insurance (ISHIP or GAIP). We encourage participants to learn more about these resources [here](https://wellbeing.uw.edu/mental-health/mental-health-resources/).
    * Faculty/Staff/Postdocs: Faculty and staff can avail short-term, focussed therapy sessions through the Washington State Employee Assistance Program ([WA EAP](https://hr.uw.edu/worklife/employee-assistance-program/)).
    * Everyone enrolled with [PEBB](https://www.hca.wa.gov/pebb-benefits-admins/pebb-benefits#:~:text=The%20Public%20Employees%20Benefits%20Board,Care%20Assistance%20Program%20(DCAP).) is eligible.
* Resources for workplace-related mental health issues relevant to geoscientists:
    * [Climate Anxiety & Eco-Grief](https://www.drjenniferatkinson.com/)
    * [climate pshchiatry alliance](https://www.climatepsychiatry.org/)
    * [Workplace mental health and well-being](https://www.hhs.gov/surgeongeneral/priorities/workplace-well-being/index.html)
    * [Work That Reconnects Network](https://www.hhs.gov/surgeongeneral/priorities/workplace-well-being/index.html)

> * Participating in supportive communities: Many of us struggle alone and have inhibtations to discuss our feelings because we might fail to identify in the same way as our collagues in the workplace. In these situations, finding communities and groups of scientists whom we can identify with personally, seek mentorship from, and rediscover purpose can be helpful. 

Reviewer: ```suggestion
* Participating in supportive and inclusive communities: Many of us struggle alone and have inhibitions when discussing our feelings because we might fail to identify in the same way as our everyday workplace colleagues. This is particularly relevant if we identify ourselves as marginalized in some form or another. In these situations, engaging with communities and groups of scientists we can identify with personally might help us seek mentorship and rediscover our purpose. Along these lines, we provide an initial list of organizations that have a core focus on inclusivity and mentorship:
```

    * [Association for Women Geoscientits](https://www.awg.org/)

>     * [Asian Americans and Pacific Islanders in Geosciences](https://aapigeosci.org/)

Reviewer: ```suggestion
    * [Asian Americans and Pacific Islanders in Geosciences](https://aapigeosci.org/)
    * [GeoLatinas](https://geolatinas.org/)
```
